theme_5/src/styles/global/blank-states.scss

```diff
@@ -0,0 +1,36 @@
+$color-blankstate: rgba($color-body-text, 0.35);
+$color-blankstate-border: rgba($color-body-text, 0.2);
+$color-blankstate-background: rgba($color-body-text, 0.1);
+
+.placeholder-svg {
+  display: block;
```

Comment:
Properties should be ordered background-color, border, display, fill, height, max-height, max-width, width